src/BenchmarkManager.py

```diff
@@ -38,6 +38,60 @@
 sns.color_palette()
 
 
+def _import_class(module_path, class_name, base_dir=None):
+    """
+    Helper function which allows to replace hard coded imports of the form
+    'import MyClass from path.to.mypkg' by calling _import_class('path.to.mypkg', 'MyClass').
+    If base_dir is specified its value will be added to python search path
+    if not not already contained in it.
+    :param module_path: python module path of the module containing the class to be imported
+    :type module_path: str
+    :param class_name: the name of the class to be imported
+    :type class_name: str
+    :rtype: classobj
+    """
+
+    #make sure that base_dir is in the search path. Otherwise
+    #the module imported here might not find its libraries.    
+    if not base_dir is None and not base_dir in sys.path:
+        logging.info(f"append to sys.path: {base_dir}")
+        sys.path.append(base_dir)
+    logging.info(f"import module {module_path}")    
+    module = importlib.import_module(module_path)
+    return vars(module)[class_name]    
+
+def _getInstanceWithSubOptions(options, class_name, *args):
```

Comment:
Can we maybe change this to snake case (`get_instance_with_sub_options`) to be more in line with the other method names?